package.json

```diff
@@ -7,12 +7,19 @@
     "@testing-library/jest-dom": "^5.11.6",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.2.2",
+    "bootstrap": "^4.5.3",
+    "jquery": "^3.5.1",
```

Comment:
what??

Reply:
`bootstrap` requires `jquery` to be installed 🤷‍♂️